lib/ord.js

```diff
@@ -47,8 +48,11 @@ const initializeAppConfig = (csn) => {
                 break;
             // TODO: should be rewritten
             case ORD_RESOURCE_TYPE.entity:
-                if (!key.includes(".texts") && keyDefinition["@ODM.entityName"]) {
-                    odmEntity.push(createEntityTypeTemplate(keyDefinition));
+                if (!key.includes(".texts")) {
+                    entities.add(key);
+                    if (keyDefinition["@ODM.entityName"]) {
+                        odmEntities.push(createEntityTypeTemplate(keyDefinition));
+                    }
```

Comment:
@Fannon @swennemers @RoshniNaveenaS @zongqichen : Does it make sense (in context of ORD doc creation) to have entities defined(and 'exposed' in service.cds file) without `@ODM.entityName` annotation?

Reply:
> @aramovic79 - you mean whether they should be exposed in the CDS export? Or if they should be documented in the ORD Document as (local) Entity Types?

I mean - whether they should be considered when constructing `apiResources`.